tests/unit/peapods/peas/test_pea_factory.py

```diff
@@ -0,0 +1,18 @@
+from jina.peapods.peas.factory import PeaFactory
+
+from jina.hubble.hubio import HubIO
+from jina.parsers import set_pea_parser
+
+
+def test_container_pea(mocker, monkeypatch):
+    mock = mocker.Mock()
+
+    def _mock_pull(self):
```

Comment:
this was not the observed problem rigth? the problem was `jinahub+docker` ?

Reply:
The problem was that we correctly pull the image name from jinahub but we don't set it in the pea args.
Therefore we can safely mock pull and check that 'jinahub+docker://DummyExecutor' becomes 'docker://jinahub/dummy_executor'

Reply:
The problem was that we correctly pull the image name from jinahub but we don't set it in the pea args.
Therefore we can safely mock pull and check that 'jinahub+docker://DummyExecutor' becomes 'docker://jinahub/dummy_executor'